### DISCUSSION TITLE
While recording through wavesurfer.js library, make the voice consistent through out recording?

### DISCUSSION THREAD

**Discussion by shahbazg4045** [katspaugh/wavesurfer.js#3349] [Q&A]
We are implementing wavesurfer library for recording. But when we play the recorded file its voice is not consistent during recording.

Please suggest me, how can I make voice consistent while recording or after recording when generate audio file the voice in the recording will be consistent when played

**Comment by katspaugh** [comment]
Try to drink some water before the recording and do some vocal exercises to keep your voice consistent.
